app/js/app.module.js

```diff
@@ -20,9 +18,10 @@
           $rootScope.$apply(function() {
             $mdBottomSheet.cancel();
             $rootScope.fullscreen = false;
-            $rootScope.$broadcast("changed_window_state");
           });
         }
+
+        $rootScope.$broadcast("changed_window_state");
```

Comment:
That line was there in the if (data.fullscreen){} block, I just moved it to apply for both cases. It's kinda pointless because there's already the llb_app event for that, but I figured that if you have your own mechanism for that already I just might use it as well.